docs/platforms/android/profiling/index.mdx

```diff
@@ -11,139 +11,250 @@ sidebar_order: 5000
   Profiling uses the Android runtime's `tracer` under the hood to sample threads. There are known issues that this `tracer` can cause crashes in certain circumstances. See this <PlatformLink to="/profiling/troubleshooting#i-see-elevated-number-of-crashes-in-the-android-runtime-when-profiling-is-activated">troubleshooting</PlatformLink> entry for more information.
 </Alert>
 
-## Enable Tracing
+## Installation
 
-Profiling depends on Sentry’s Tracing product being enabled beforehand. To enable tracing in the SDK:
+Android profiling is available starting in SDK version `8.7.0`.
+The legacy profiler is available on SDK versions `6.16.0` and higher through <PlatformLink to="/configuration/options/#legacy-profiling-options">its options</PlatformLink>.
 
-In `AndroidManifest.xml`:
+## Enabling Profiling
```

Comment:
We call the new profiling product "UI Profiling" so we should use that naming here and in other places

Reply:
@ArthurKnaus or @JonasBa we should update [node](https://docs.sentry.io/platforms/javascript/guides/node/profiling/node-profiling/#enabling-profiling) to `Enabling UI Profiling`, then

Reply:
Node.js is Continuous Profiling, Android is UI Profiling. It depends on whether its a backend or frontend platform.